### PR TITLE
Add keda-manager build job

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Changes proposed in this pull request:

- Add keda-manager build job

**Related issue(s)**

See also: https://github.com/kyma-project/keda-manager/issues/5

### CODE REVIEW

File: templates/data/generic_external_module_data.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        jobConfigs:
+          - repoName: "github.com/kyma-project/keda-manager"
+            jobs:
+              - jobConfig:
+                  path: operator
+                  args:
+                    - "/home/prow/go/src/github.com/kyma-project/keda-manager/operator"
+                  run_if_changed: "^operator"
+                  optional: "true"
+                  release_since: "2.6"
```
Comment (m00g3n) [inline templates/data/generic_external_module_data.yaml:14]: change to 2.8 before commit

**Review by pPrecel — APPROVED** [review]

**Comment by pPrecel** [comment]
why is this pipeline empty? 🤔 
https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_keda-manager/9/m00g3n_test_of_prowjob_pre-keda-manager-operator-lint/1571892481983778816

**Comment by m00g3n** [comment]
> why is this pipeline empty? 🤔 https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_keda-manager/9/m00g3n_test_of_prowjob_pre-keda-manager-operator-lint/1571892481983778816
 
it's lint - it will be non-empty if it would fail;
as we agreed offline I've updated the command not to produce blank output

File: templates/data/generic_external_module_data.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    - jobConfig_generic_component
+                    - "build_labels" # default labels
+              - jobConfig:
+                  labels:
+                    preset-docker-push-repository-kyma: "true" # sets GCR Registry through DOCKER_PUSH_REPOSITORY
+                  name: post-keda-manager-operator-build # "post-main-lifecycle-manager-build"
+                  always_run: true
+                  command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build-generic.sh"
+                  args:
+                    - "/home/prow/go/src/github.com/kyma-project/keda-manager/operator"
```
Comment (Ressetkk) [inline templates/data/generic_external_module_data.yaml:67]: Please refrain from using `build-generic.sh` and use `kaniko-build` image for building images. We aim to remove that script.

Look at new jobs for lifecycle-manager that were recently updated. If you need any help, then just reach me out.

**Review by pPrecel — DISMISSED** [review]

**Review by Halamix2 — APPROVED** [review]

**Pull request merged**